### PR TITLE
Import CPPABI64 into this repository.

### PR DESCRIPTION
**Pull request by statham-arm**

### CODE REVIEW

File: cppabi64/cppabi64.rst [hunk 1]
```diff
@@ -0,0 +1,778 @@
+..
+   Copyright (c) 2003-2020, Arm Limited and its affiliates.  All rights reserved.
```
Comment (stuij) [inline cppabi64/cppabi64.rst:2]: this should read: 2010-2021

File: cppabi64/cppabi64.rst [hunk 2]
```diff
@@ -0,0 +1,778 @@
...
+   CC-BY-SA-4.0 AND Apache-Patent-License
+   See LICENSE file for details
+
+.. |release| replace:: 2021Q1
+.. |date-of-issue| replace:: 18\ :sup:`th` February 2021
+.. |copyright-date| replace:: 2003-2021
```
Comment (stuij) [inline cppabi64/cppabi64.rst:8]: this should read: 2010-2021

File: cppabi64/cppabi64.rst [hunk 3]
```diff
@@ -0,0 +1,778 @@
...
+
+The pointer to member function representation differs from that used
+by Itanium. See `Representation of pointer to member function`_.
+
+**GC++ABI §**\ 2.8_ **Initialization guard variables**
+
+This ABI only specifies the bottom bit of the guard variable. See
+`Guard variables`_.
+
+**GC++ABI §**\ 3.1.3_ **Return Values**
```
Comment (stuij) [inline cppabi64/cppabi64.rst:453]: ah nice, I see these paragraph numbers have changed

File: cppabi64/cppabi64.rst [hunk 4]
```diff
@@ -0,0 +1,778 @@
...
+
+Unwind Table Location
+^^^^^^^^^^^^^^^^^^^^^
+
+Exception tables are located in sections with the name ``.eh_frame``
+and ``.eh_frame_hdr``. Linkers shall put the ``.eh_frame_hdr`` section
+in a single text segment, with a ``PT_AARCH64_UNWIND`` program table
+entry identifying the unwind table header location.
+
+
```
Comment (stuij) [inline cppabi64/cppabi64.rst:689]: missing page break

File: cppabi64/cppabi64.rst [hunk 5]
```diff
@@ -0,0 +1,778 @@
...
+The unwind library may elect to transfer control to landing pads via a jump
+instruction that requires the jump target to be identified as a valid
+destination for indirect jumps.
+
+.. note::
+   The intent is that the landing pads should start with a ``bti j`` instruction
+   (or equivalent), when they might operate in an environment with Branch Target
+   Identification enabled.
+
```
Comment (stuij) [inline cppabi64/cppabi64.rst:708]: missing page break

File: cppabi64/cppabi64.rst (hunk 2 above)
Comment (statham-arm) [reply cppabi64/cppabi64.rst:8]: How on earth did I manage that? I guess I must have been overenthusiastic about using CPPABI32 as a model for the overall document structure. Oops.

**Comment by stuij** [comment]
LGTM. Thanks!

**Pull request merged**